### DISCUSSION TITLE
is it possible to use fltk with glow?

### DISCUSSION THREAD

**Discussion by GodGotzi** [fltk-rs/fltk-rs#1504] [Q&A]
I need it for a project of mine to combine a 3d perspective with the fltk gui, I already have the code for the 3d view. I worked with egui so far in my project, but now I consider switching to fltk, because an event driven system is more useful in my usecase. So my question is there any possible way to render fltk with glow (gl)?

**Comment by MoAlyousef** [comment]
Hi
Yes it's possible. Here you can find a demo program:
https://github.com/fltk-rs/demos/tree/master/glow

Also a recent relevant discussion:
https://github.com/fltk-rs/fltk-rs/discussions/1495
